AdyenComponents/Apple Pay/ApplePayComponent.swift

```diff
@@ -9,48 +9,23 @@ import Foundation
 import PassKit
 
 /// A component that handles Apple Pay payments.
-public class ApplePayComponent: NSObject, PaymentComponent, PresentableComponent, Localizable, LoadingComponent {
+public class ApplePayComponent: NSObject, PresentableComponent, PaymentComponent, Localizable, FinalizableComponent {
+
+    /// The Apple Pay payment method.
+    public var paymentMethod: PaymentMethod {
+        configuration.paymentMethod
+    }
 
     /// Apple Pay component configuration.
     internal let configuration: Configuration
-
-    /// Flag to indicate that component wasn't stopped by the merchant's code.
-    /// By default all cancelations assumed to be initiated by the user.
-    /// :nodoc:
-    internal var isUserCancel = true
-    internal var paymentAuthorizationCompletion: ((PKPaymentAuthorizationStatus) -> Void)?
     internal var paymentAuthorizationViewController: PKPaymentAuthorizationViewController?
+    internal var paymentAuthorizationCompletion: ((PKPaymentAuthorizationStatus) -> Void)?
     
     /// The delegate of the component.
     public weak var delegate: PaymentComponentDelegate?
     
-    /// The Apple Pay payment method.
-    public var paymentMethod: PaymentMethod {
-        configuration.paymentMethod
-    }
-    
-    /// The line items for this payment.
-    public var summaryItems: [PKPaymentSummaryItem] {
-        configuration.summaryItems
-    }
-    
-    /// A list of fields that you need for a billing contact in order to process the transaction.
-    /// Ignored on iOS 10.*.
-    public var requiredBillingContactFields: Set<PKContactField> {
-        configuration.requiredBillingContactFields
-    }
-    
-    /// A list of fields that you need for a shipping contact in order to process the transaction.
-    /// Ignored on iOS 10.*.
-    public var requiredShippingContactFields: Set<PKContactField> {
-        configuration.requiredShippingContactFields
-    }
-    
     /// Initializes the component.
     ///
-    /// - Warning: `stopLoading()` must be called before dismissing this component.
```

Comment:
maybe change it to `/// - Warning: `didFInalize()` must be called before dismissing this component.`

Reply:
👍